plugins/fpc/fu-fpc-device.c

```diff
@@ -224,8 +225,10 @@ fu_fpc_device_setup_version(FuFpcDevice *self, GError **error)
 						   0,
 						   FALSE,
 						   FALSE,
-						   error))
+						   error)) {
+				g_prefix_error(error, "fu_fpc_device_setup_version: fail to clear status: ");
```

Comment:
please don't put the `fu_fpc_device_setup_version: ` prefix in error messages

Reply:
OK